portal/src/main/webapp/WEB-INF/jsp/oncoprint/main.jsp

```diff
@@ -43,172 +43,18 @@
     var showBinaryCustomDriverAnnotation = "<%=(boolean) GlobalProperties.showBinaryCustomDriverAnnotation()%>";
     var showTiersCustomDriverAnnotation = "<%=(boolean) GlobalProperties.showTiersCustomDriverAnnotation()%>";
     
+    //var hasDriverAnnotations = "<%=(boolean) DaoMutation.hasDriverAnnotations(request.getParameter(QueryBuilder.CANCER_STUDY_ID))%>" === "true";
+    //var customDriverTiers = "<%=String.join(",", DaoMutation.getTiers(request.getParameter(QueryBuilder.CANCER_STUDY_ID)))%>".split(",");
```

Comment:
Where do we get these 2 values now? Inside the react component?

Reply:
we're just not going to address that for this release